lemarche/www/tenders/urls.py

```diff
@@ -1,11 +1,13 @@
 from django.urls import path
 
-from lemarche.www.tenders.views import TenderAddView
+from lemarche.www.tenders.views import TenderAddView, TenderDetail, TenderListView
 
 
 # https://docs.djangoproject.com/en/dev/topics/http/urls/#url-namespaces-and-included-urlconfs
 app_name = "tenders"
 
 urlpatterns = [
     path("ajouter", TenderAddView.as_view(), name="add"),
```

Comment:
```suggestion
    path("ajouter", TenderCreateView.as_view(), name="create"),
```

c'est subtil, mais j'ai appliqué cette règle sur les autres vues, pour mimick la nomenclature de Django Class Based Views (List, Detail, Create, Update, Delete)